include/wx/peninfo.h

```diff
@@ -0,0 +1,145 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/peninfo.h
```

Comment:
I get errors in peninfo.h that all the wxPen constants aren't defined (wxPenStyle, etc.), even though I added an include for pen.h. What is the proper way of including pen.h here?